content/5_Plat/Geo_Pri.mdx

```diff
@@ -384,6 +384,207 @@ int main() {
 </CPPSection>
 </LanguageSection>
 
+### Euclidean minimum spanning tree - EMST
+
+<FocusProblem problem="emst"/>
+
+#### Initial Approach
+
+The problem is a combination of EMST - euclidean minimum spanning tree - and LCA. We can compute the EMST using Kruskal's or Prim's algorithm. Using this approach, the algorithm
+will run on $n^2$ edges, which causes TLE.
```

Comment:
> Using this approach, the algorithm will run on $n^2$ edges, which causes TLE.

so does this sol work

Reply:
It works, but id gets TLE

Reply:
uh then what's the point of this solution

Reply:
OH I JUST SAW